.github/workflows/_triage.yaml

```diff
@@ -117,8 +117,8 @@ jobs:
           filter_valid_tags() {
             fgrep -x -f <(echo "$ALL_BASE_TAGS")
           }
-          start_date=${BASE_IMAGE##*nightly-}
-          end_date=$(get_build_date ${{ secrets.GITHUB_TOKEN }} $BROKEN_IMAGE )
+          start_date=$(get_build_date ${{ secrets.GITHUB_TOKEN }} $BASE_IMAGE linux amd64)
```

Comment:
Please use [the `docker-login` action](https://github.com/marketplace/actions/docker-login) as a job step to authenticate to the registry, and then have the individual scripts assume that the docker CLI is already properly granted access to the given image.

Reply:
We already use that action and sometimes use the `docker` CLI. 
See https://github.com/NVIDIA/JAX-Toolbox/pull/315#discussion_r1363562867 for the case that seems non-trivial to cover with `docker`.